runtime/ioext/ioext.go

```diff
@@ -0,0 +1,24 @@
+package ioext
+
+import (
+	"io"
+)
+
+// ReadSeekCloser implements io.Reader, io.Seeker, and io.Closer. It is trivially implemented by os.File.
+type ReadSeekCloser interface {
+	io.Reader
+	io.Seeker
+	io.Closer
+}
+
+func NopCloser(r io.Reader) ReadSeekNopCloser {
```

Comment:
Haha, yes. I'll circle back and get these things before we do a real review. Should've defined "30% review" before I asked for one. I just use it to mean something between a design review and checking to see if everything is documented correctly :smile:. There's still a great bit of work to do before this is ready to be landed.